packages/web-components/src/components/rux-input/rux-input.tsx

```diff
@@ -127,17 +134,30 @@ export class RuxInput implements FormFieldInterface {
         this._handleSlotChange()
     }
 
+    //! Tried this to fix the type not changing correctly in SB. This never even fires.
+    @Watch('type')
```

Comment:
In SB default input story, changing the type to password doesn't do anything, and this watch never fires. I've got an example using a button and js in index.html as well. What's weird is the class updates - like if you switch type to search, it will update the class, but not fire the watch.